### PR TITLE
dev-libs/ffcall: Fix the build with slibtool

### PR DESCRIPTION
**Pull request by orbea**
Bug: https://bugs.gentoo.org/776976
Upstream-ML: https://lists.gnu.org/archive/html/libffcall/2022-07/msg00000.html

### CODE REVIEW

File: dev-libs/ffcall/files/ffcall-2.4-slibtool.patch [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Bug: https://bugs.gentoo.org/776976
```
Comment (thesamesam) [inline dev-libs/ffcall/files/ffcall-2.4-slibtool.patch:1]: Is this upstream somewhre?

Comment (orbea) [reply dev-libs/ffcall/files/ffcall-2.4-slibtool.patch:1]: I tried to send a patch to the mailing list, but I don't see it yet.

https://lists.gnu.org/archive/html/libffcall/

Comment (orbea) [reply dev-libs/ffcall/files/ffcall-2.4-slibtool.patch:1]: I see it now.

https://lists.gnu.org/archive/html/libffcall/2022-07/msg00000.html

Comment (thesamesam) [reply dev-libs/ffcall/files/ffcall-2.4-slibtool.patch:1]: Thanks, I'll throw the link in.

Comment (thesamesam) [reply dev-libs/ffcall/files/ffcall-2.4-slibtool.patch:1]: oh, you already did :)

**Pull request closed without merging**